AWSLex.podspec

```diff
@@ -18,42 +18,5 @@ Pod::Spec.new do |s|
   s.private_header_files = 'AWSLex/Bluefront/include/*.h'
   s.preserve_paths = 'AWSLex/Bluefront/include/*'
   s.vendored_libraries = 'AWSLex/Bluefront/libBlueAudioSourceiOS.a'
-  s.resource_bundle = { 'AWSLex' => 'AWSLex/Media.xcassets' }
```

Comment:
Was this manually added before?

Reply:
moving to draft, need to investigate previous AWSLex model updates and compare

Reply:
We'll investigate the required codegen changes, but in the meantime we'll merge this in.